### ISSUE TITLE
Test LightGBM based on XGboost Operator

### ISSUE THREAD

**Issue by merlintang** [kubeflow/xgboost-operator#56]
LightGBM is also popular for the boosting tree model, thus, it is natural to demo or test how LGBM to run over XGboost Operator.

**Comment by merlintang** [comment]
@XinkaiWang515 and @ xfate123

**Comment by terrytangyuan** [comment]
Hmm this is XGBoost operator though. We should probably aim to provide better supporting experience for XGBoost users. I don’t think LightGBM would need to be supported here.

**Comment by merlintang** [comment]
Yes, I totally agree that we need to provide better user experience for
XGBoost. For example, better Python SDK.

we can keep in mind that XGboost operator could be for any boosting tree
related algorithm. Now, more and more groups are using LGBM. We can leave
this into minor priority.

**Comment by merlintang** [comment]
Leave this for further work.
